src/app/health-worker/children/[childId]/page.tsx

```
@@ -0,0 +1,362 @@
+        <div className="rounded-2xl border border-red-200 bg-red-50 p-8 text-center">
+          <XCircle className="mx-auto size-10 text-red-400" />
+          <h2 className="mt-3 text-lg font-semibold text-red-800">Child Not Found</h2>
+          <p className="mt-1 text-sm text-red-600">The requested child record could not be loaded.</p>
+        </div>
+      </div>
+    );
+  }
+
+  const statusCfg = HEALTH_STATUS_CONFIG[child.health_status];
+  const StatusIcon = statusCfg.icon;
+  const ageMonths = Math.floor((Date.now() - new Date(child.date_of_birth).getTime()) / (30.44 * 24 * 60 * 60 * 1000));
```

ESLint

[error] Error: Cannot call impure function during render

`Date.now` is an impure function. Calling an impure function can produce unstable results that update unpredictably when the component happens to re-render. (https://react.dev/reference/rules/components-and-hooks-must-be-pure#components-and-hooks-must-be-idempotent).

/home/jailuser/git/src/app/(health-worker)/children/[childId]/page.tsx:249:33
  247 |   const statusCfg = HEALTH_STATUS_CONFIG[child.health_status];
  248 |   const StatusIcon = statusCfg.icon;
> 249 |   const ageMonths = Math.floor((Date.now() - new Date(child.date_of_birth).getTime()) / (30.44 * 24 * 60 * 60 * 1000));
      |                                 ^^^^^^^^^^ Cannot call impure function
  250 |   const ageLabel = ageMonths < 12 ? `${ageMonths}mo` : `${Math.floor(ageMonths / 12)}y ${ageMonths % 12}mo`;
  251 |   const administered = child.vaccinations.filter((v) => v.status === "ADMINISTERED").length;
  252 |   const overdue = child.vaccinations.filter((v) => v.status === "OVERDUE").length;

(react-hooks/purity)